osu.Game/Rulesets/Scoring/Score.cs

```diff
@@ -28,9 +29,21 @@ public class Score
         public int Combo { get; set; }
 
         [JsonProperty(@"mods")]
-        protected string[] ModStrings { get; set; } //todo: parse to Mod objects
+        private string[] modStrings { get; set; }

```

Comment:
Have you tested that this is actually parsed from web? Last I remember private fields/properties weren't decoded.